backend/functions/lib/adapters/gateways/orders.cosmosdb.repository.ts

```diff
@@ -62,16 +74,17 @@ export class OrdersCosmosDbRepository implements OrdersRepository {
         .database(this.cosmosConfig.databaseName)
         .container(this.containerName)
         .item(id, data.caseId)
-        .read();
+        .read<TransferOrder>();
```

Comment:
Should this be Order rather than TransferOrder?

Reply:
Tried it.  It didn't seem to make any difference.  App still runs fine and tests passed.

Reply:
So it didn't seem to make a difference but I later realized that the orders.consmosdb.repository.ts file is only for Transfers.  Consolidations has it's own repo.  We should rename this repository file.  That was a waste of my time.

Reply:
We may want to rename orders => transfers / transferOrders. Not with CAMS-270.